src/character.cpp

```diff
@@ -146,13 +150,18 @@ double Character::aim_per_move( const item& gun, double recoil ) const
         return 0; // no suitable sights (already at maxium aim)
     }
 
+    // each 5 points (combined) of hand encumbrance increases aim cost by one unit
+    cost += round ( ( encumb( bp_arm_l ) + encumb( bp_arm_r ) ) / 10.0 );
+
+    ///\EFFECT_DEX increases aiming speed
+    cost += 8 - dex_cur;
```

Comment:
I'm improving the test harness to produce some graphs from that, give me a moment and we can get some data on that.